compss/runtime/scripts/utils/chameleon_cluster_setup

```diff
@@ -1,190 +1,190 @@
-echo "Type  1 if connected via fabnetv4 network otherwise type 2 if connected via sharednet1"
-read choice
-if [ "$choice" == "1" ]; then
-   NETWORK="fabnetv4"
-else
-   NETWORK="sharednet1"
-fi
-echo " "
-
-##########################################################
-# Retrieve other information
-echo "* Retrieving configuration parameters from Chameleon..."
-image=$(openstack server show "$masterName" -f value -c image | awk '{print $2}' | sed 's/[()]//g')
-netId=$(openstack network list | grep "$NETWORK" | awk '{print $2}')
-
-##########################################################
-# Launch workers 
-echo "* Launching workers..."
-# Insert COMPSs Master key to OpenStack. Create workers with COMPSsMaster key authorized
-openstack keypair create --public-key /home/cc/.ssh/id_rsa.pub COMPSsMaster${randomID}
-
-# Create workers
-for (( i=1; i<=numWorkers; i++ )); do
-  cmd="openstack server create --flavor baremetal --image $image --key-name COMPSsMaster${randomID} --nic net-id=$netId --hint reservation=$reservationId COMPSsWorker$i"
-  echo "$cmd"
-  $cmd
-  sleep $SLEEP_BETWEEN_WORKER_CREATION
-done
-
-##########################################################
-# Waiting Workers creation
-echo "* Waiting for COMPSs Workers creation (eta 5 min)"
-for (( i=1; i<=instanceCreationTime; i++ )); do
-  printf "."
-  sleep ${HALF_MIN}
-done
-printf "\\n"
-
-for (( i=1; i<=numWorkers; i++ )); do
-  # Wait for each worker
-  cmd_status=$(openstack server list | grep "COMPSsWorker$i" | awk '{print $6}')
-  while [ "$cmd_status" != "ACTIVE" ]; do
+  #!/bin/bash -e
+
+  #=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+  #
+  # Script Name: chameleon_cluster_setup
+  # Author:      COMPSs Support <support-compss@bsc.es>
+  #              Barcelona SuperComputing Center (BSC-CNS)
+  #
+  # Description: Script to setup a COMPSs cluster within a
+  #              Chameleon lease
+  #
+  #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+
+  # Setting up COMPSS_HOME
+  export COMPSS_HOME="/opt/COMPSs/"
+
+  ##########################################################
+  # Script variables
+  user=cc
+  instanceCreationTime=10        # Iterations over 30s
+  sshUpTime=8                    # Iterations over 30s
+  randomID=$RANDOM
+  tmpFile=/tmp/compss-workers-${randomID}.tmp
+  HALF_MIN=30s
+  SLEEP_BETWEEN_WORKER_CREATION=5
+
+  ##########################################################
+  # Prompt warning message
+  echo "!! WARNING: Remember to source the OpenStack RC File"
+  echo " "
+  sleep 2s
+
+  # Prompt messages to get information
+  read -rp "Provide the name of the COMPSs Master Instance (this instance): " masterName
+  read -rp "Provide the reservation ID to deploy COMPSs: " reservationId
+  read -rp "Provide the number of COMPSs Workers: " numWorkers
+  echo "Type  1 if connected via fabnetv4 network otherwise type 2 if connected via sharednet1"
+  read choice
+  if [ "$choice" == "1" ]; then
+    NETWORK="fabnetv4"
```

Comment:
We need to verify if it is ok to use this network, before merging

Reply:
It was mostly for my own convinience during testing [CHI@UC], in previous script also it would just detect the name of network used and just use it without restrictions on fabnet4

Reply:
I have updated the commit such that there is no option for fabnetv4 and it directly takes sharednet1, so there shoud be no problem now.